query-input/src/main/java/de/brandwatch/minianalytics/queryinput/controller/RestController.java

```diff
@@ -0,0 +1,65 @@
+package de.brandwatch.minianalytics.queryinput.controller;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import de.brandwatch.minianalytics.queryinput.postgres.model.Query;
+import de.brandwatch.minianalytics.queryinput.postgres.repository.QueryRepository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+
+
+@org.springframework.web.bind.annotation.RestController
+public class RestController {
+
+    Logger logger = LoggerFactory.getLogger(RestController.class);
+
+    @Autowired
+    QueryRepository queryRepository;
+
+    static final private Gson gson = new GsonBuilder().setPrettyPrinting().create();
```

Comment:
the normal order would be `private static final`

Reply:
Done